addons/binding/pom.xml

```diff
@@ -24,6 +24,7 @@
     <module>org.openhab.binding.hdanywhere</module>
     <module>org.openhab.binding.ipp</module>
     <module>org.openhab.binding.keba</module>
+    <module>org.openhab.binding.kostal.inverter</module>
```

Comment:
we do not use dots within the binding name - so I would prefer changing it to `org.openhab.binding.kostalinverter`.